*** tests/A-01-05-M-ok.out	2019-11-15 11:49:36.401106651 +0000
--- tests/A-01-05-M-ok.outhyp	2019-12-06 18:53:08.905238970 +0000
***************
*** 14,20 ****
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: 0 - Obi-Wan Kenobi - obiwan@jedi.org - NORMAL - ACTIVO
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
--- 14,20 ----
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: 0 - Obi-Wan Kenobi - obiwan@jedi.org - CUMPRIDOR - ACTIVO
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
