*** tests/A-04-03-M-ok.out	2019-11-13 10:10:45.280583483 +0000
--- tests/A-04-03-M-ok.outhyp	2019-11-13 15:30:19.675068998 +0000
***************
*** 12,24 ****
  2 - Mostrar todas as obras
  3 - Efectuar pesquisa de termos
  0 - Sair
! Escolha uma opção: Introduza o número da obra: 0 - 22 de  22 - Livro - A arte de sobreviver no 36 - 30 - Ficção - Joao Fonseca - 1234567892
  Menu de obras
  1 - Mostrar obra
  2 - Mostrar todas as obras
  3 - Efectuar pesquisa de termos
  0 - Sair
! Escolha uma opção: Introduza o número da obra: 5 - 50 de  50 - Livro - Dicionário de Programação - 20 - Referência - Odete Fonseca - 1234567890
  Menu de obras
  1 - Mostrar obra
  2 - Mostrar todas as obras
--- 12,24 ----
  2 - Mostrar todas as obras
  3 - Efectuar pesquisa de termos
  0 - Sair
! Escolha uma opção: Introduza o número da obra: 0 - 22 de 22 - Livro - A arte de sobreviver no 36 - 30 - FICTION - Joao Fonseca - 1234567892
  Menu de obras
  1 - Mostrar obra
  2 - Mostrar todas as obras
  3 - Efectuar pesquisa de termos
  0 - Sair
! Escolha uma opção: Introduza o número da obra: 5 - 50 de 50 - Livro - Dicionário de Programação - 20 - REFERENCE - Odete Fonseca - 1234567890
  Menu de obras
  1 - Mostrar obra
  2 - Mostrar todas as obras
