*** tests/A-04-03-M-ok.out	2019-11-13 10:10:45.280583483 +0000
--- tests/A-04-03-M-ok.outhyp	2019-11-13 11:56:09.279700611 +0000
***************
*** 18,24 ****
  2 - Mostrar todas as obras
  3 - Efectuar pesquisa de termos
  0 - Sair
! Escolha uma opção: Introduza o número da obra: 5 - 50 de  50 - Livro - Dicionário de Programação - 20 - Referência - Odete Fonseca - 1234567890
  Menu de obras
  1 - Mostrar obra
  2 - Mostrar todas as obras
--- 18,24 ----
  2 - Mostrar todas as obras
  3 - Efectuar pesquisa de termos
  0 - Sair
! Escolha uma opção: Introduza o número da obra: Mostrar obra: Operação inválida: A obra 5 não existe.
  Menu de obras
  1 - Mostrar obra
  2 - Mostrar todas as obras
