*** tests/A-07-07-M-ok.out	2019-10-28 12:43:38.000000000 +0000
--- tests/A-07-07-M-ok.outhyp	2019-11-13 00:12:21.323594729 +0000
***************
*** 7,14 ****
  6 - Menu de gestão de obras
  7 - Menu de gestão de requisições
  0 - Sair
! Escolha uma opção: Ficheiro a abrir: Abrir dados: Operação inválida: O ficheiro 'naoExistente' não existe.
! Menu Principal
  1 - Abrir dados
  2 - Guardar dados
  3 - Ver data
--- 7,13 ----
  6 - Menu de gestão de obras
  7 - Menu de gestão de requisições
  0 - Sair
! Escolha uma opção: Ficheiro a abrir: Menu Principal
  1 - Abrir dados
  2 - Guardar dados
  3 - Ver data
