*** tests/A-07-04-M-ok.out	2019-11-20 22:20:39.000000000 +0000
--- tests/A-07-04-M-ok.outhyp	2019-12-06 18:53:13.531550974 +0000
***************
*** 23,29 ****
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: Introduza o número de utente: 0 - Obi-Wan Kenobi - obiwan@jedi.org - NORMAL - ACTIVO
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
--- 23,29 ----
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: Introduza o número de utente: 0 - Obi-Wan Kenobi - obiwan@jedi.org - CUMPRIDOR - ACTIVO
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
