<<<<<<< HEAD
*** tests/A-09-02-M-ok.out	2019-11-20 22:21:27.000000000 +0000
--- tests/A-09-02-M-ok.outhyp	2019-12-06 11:01:22.305003763 +0000
=======
*** tests/A-09-02-M-ok.out	2019-12-03 17:06:33.556738066 +0000
--- tests/A-09-02-M-ok.outhyp	2019-12-06 10:44:27.352010448 +0000
>>>>>>> afcaf0b30256a4011b43884ed9af86e5eb926ce8
***************
*** 60,67 ****
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: Introduza o número de utente: ENTREGA: 7 - 1 de  1 - DVD - The rise of Skywalker - 20 - Técnica e Científica - J. J. Abrams - 200505551
! Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
  3 - Listar utentes
--- 60,66 ----
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
  3 - Listar utentes
