*** tests/A-12-01-M-ok.out	2019-11-20 22:26:39.000000000 +0000
--- tests/A-12-01-M-ok.outhyp	2019-12-04 18:51:05.706531950 +0000
***************
*** 11,17 ****
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: Devolver obra: Operação inválida: A obra 11 não existe.
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
--- 11,17 ----
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: Devolver obra: Operação inválida: A obra 11 não foi requisitada pelo utente 1.
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
