*** tests/A-12-04-M-ok.out	2019-12-05 10:45:14.000000001 +0000
--- tests/A-12-04-M-ok.outhyp	2019-12-05 12:02:11.000000001 +0000
***************
*** 38,44 ****
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
--- 38,46 ----
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: -1
! Devolver obra: Operação inválida: A obra 1 não foi requisitada pelo utente 1.
! Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
