<<<<<<< HEAD
*** tests/A-12-05-M-ok.out	2019-11-20 22:26:39.000000000 +0000
--- tests/A-12-05-M-ok.outhyp	2019-12-06 12:25:04.592001000 +0000
=======
*** tests/A-12-05-M-ok.out	2019-12-03 17:06:33.560740065 +0000
--- tests/A-12-05-M-ok.outhyp	2019-12-06 12:04:59.797002548 +0000
>>>>>>> fea8a80e6369c21a5415e04883daeb02326cb49b
***************
*** 20,26 ****
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: A obra 1 deve ser devolvida no dia 22.
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
--- 20,26 ----
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: A obra 1 deve ser devolvida no dia 28.
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
