*** tests/A-13-02-M-ok.out	2019-12-11 22:15:32.845877476 +0000
--- tests/A-13-02-M-ok.outhyp	2019-12-12 16:20:44.687651186 +0000
***************
*** 70,76 ****
  5 - Saldar multa de utente
  0 - Sair
  Escolha uma opção: 0 - Chewbacca - chewbacca@milenium.falcon.org - NORMAL - ACTIVO
! 1 - Darth Vader - darth@dark.force.coAm - NORMAL - SUSPENSO - EUR 85
  2 - Luke Skywalker - last.jedi@universe.com - NORMAL - ACTIVO
  3 - Rey - real.last.jedi@universe.com - NORMAL - ACTIVO
  Menu de Gestão de Utentes
--- 70,76 ----
  5 - Saldar multa de utente
  0 - Sair
  Escolha uma opção: 0 - Chewbacca - chewbacca@milenium.falcon.org - NORMAL - ACTIVO
! 1 - Darth Vader - darth@dark.force.coAm - FALTOSO - SUSPENSO - EUR 85
  2 - Luke Skywalker - last.jedi@universe.com - NORMAL - ACTIVO
  3 - Rey - real.last.jedi@universe.com - NORMAL - ACTIVO
  Menu de Gestão de Utentes
***************
*** 88,94 ****
  5 - Saldar multa de utente
  0 - Sair
  Escolha uma opção: 0 - Chewbacca - chewbacca@milenium.falcon.org - NORMAL - ACTIVO
! 1 - Darth Vader - darth@dark.force.coAm - NORMAL - ACTIVO
  2 - Luke Skywalker - last.jedi@universe.com - NORMAL - ACTIVO
  3 - Rey - real.last.jedi@universe.com - NORMAL - ACTIVO
  Menu de Gestão de Utentes
--- 88,94 ----
  5 - Saldar multa de utente
  0 - Sair
  Escolha uma opção: 0 - Chewbacca - chewbacca@milenium.falcon.org - NORMAL - ACTIVO
! 1 - Darth Vader - darth@dark.force.coAm - FALTOSO - ACTIVO
  2 - Luke Skywalker - last.jedi@universe.com - NORMAL - ACTIVO
  3 - Rey - real.last.jedi@universe.com - NORMAL - ACTIVO
  Menu de Gestão de Utentes
