*** tests/A-13-09-M-ok.out	2019-12-11 22:15:32.845877476 +0000
--- tests/A-13-09-M-ok.outhyp	2019-12-12 16:46:03.422639600 +0000
***************
*** 59,65 ****
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: Introduza o número de utente: 1 - Darth Vader - darth@dark.force.coAm - NORMAL - SUSPENSO - EUR 10
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
--- 59,65 ----
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: Introduza o número de utente: 1 - Darth Vader - darth@dark.force.coAm - FALTOSO - SUSPENSO - EUR 10
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
***************
*** 92,98 ****
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: Introduza o número de utente: 1 - Darth Vader - darth@dark.force.coAm - NORMAL - SUSPENSO - EUR 10
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
--- 92,98 ----
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: Introduza o número de utente: 1 - Darth Vader - darth@dark.force.coAm - FALTOSO - SUSPENSO - EUR 10
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
