*** tests/A-13-01-M-ok.out	2019-11-20 22:26:59.000000000 +0000
--- tests/A-13-01-M-ok.outhyp	2019-12-06 14:00:04.962142936 +0000
***************
*** 20,26 ****
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: A obra 1 deve ser devolvida no dia 22.
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
--- 20,26 ----
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: A obra 1 deve ser devolvida no dia 28.
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
***************
*** 62,77 ****
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: O utente 1 deve pagar uma multa de EUR 90.
! O utente deseja pagar multa (s/n)? Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: O utente 2 deve pagar uma multa de EUR 25.
! O utente deseja pagar multa (s/n)? Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
  Escolha uma opção: Menu Principal
  1 - Abrir dados
  2 - Guardar dados
--- 62,77 ----
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Número inválido!
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
+ Escolha uma opção: Número inválido!
  Escolha uma opção: Menu Principal
  1 - Abrir dados
  2 - Guardar dados
***************
*** 89,96 ****
  5 - Saldar multa de utente
  0 - Sair
  Escolha uma opção: 0 - Chewbacca - chewbacca@milenium.falcon.org - NORMAL - ACTIVO
! 1 - Darth Vader - darth@dark.force.coAm - FALTOSO - SUSPENSO - EUR 90
! 2 - Luke Skywalker - last.jedi@universe.com - NORMAL - SUSPENSO - EUR 25
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
--- 89,96 ----
  5 - Saldar multa de utente
  0 - Sair
  Escolha uma opção: 0 - Chewbacca - chewbacca@milenium.falcon.org - NORMAL - ACTIVO
! 1 - Darth Vader - darth@dark.force.coAm - NORMAL - ACTIVO
! 2 - Luke Skywalker - last.jedi@universe.com - NORMAL - ACTIVO
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
***************
*** 98,104 ****
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
  3 - Listar utentes
--- 98,105 ----
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Saldar multa de utente: Operação inválida: O utente 1 não se encontra suspenso.
! Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
  3 - Listar utentes
***************
*** 106,113 ****
  5 - Saldar multa de utente
  0 - Sair
  Escolha uma opção: 0 - Chewbacca - chewbacca@milenium.falcon.org - NORMAL - ACTIVO
! 1 - Darth Vader - darth@dark.force.coAm - FALTOSO - ACTIVO
! 2 - Luke Skywalker - last.jedi@universe.com - NORMAL - SUSPENSO - EUR 25
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
--- 107,114 ----
  5 - Saldar multa de utente
  0 - Sair
  Escolha uma opção: 0 - Chewbacca - chewbacca@milenium.falcon.org - NORMAL - ACTIVO
! 1 - Darth Vader - darth@dark.force.coAm - NORMAL - ACTIVO
! 2 - Luke Skywalker - last.jedi@universe.com - NORMAL - ACTIVO
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
***************
*** 115,121 ****
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
  3 - Listar utentes
--- 116,123 ----
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Saldar multa de utente: Operação inválida: O utente 2 não se encontra suspenso.
! Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
  3 - Listar utentes
***************
*** 123,129 ****
  5 - Saldar multa de utente
  0 - Sair
  Escolha uma opção: 0 - Chewbacca - chewbacca@milenium.falcon.org - NORMAL - ACTIVO
! 1 - Darth Vader - darth@dark.force.coAm - FALTOSO - ACTIVO
  2 - Luke Skywalker - last.jedi@universe.com - NORMAL - ACTIVO
  Menu de Gestão de Utentes
  1 - Registar utente
--- 125,131 ----
  5 - Saldar multa de utente
  0 - Sair
  Escolha uma opção: 0 - Chewbacca - chewbacca@milenium.falcon.org - NORMAL - ACTIVO
! 1 - Darth Vader - darth@dark.force.coAm - NORMAL - ACTIVO
  2 - Luke Skywalker - last.jedi@universe.com - NORMAL - ACTIVO
  Menu de Gestão de Utentes
  1 - Registar utente
