*** tests/A-13-07-M-ok.out	2019-12-11 22:15:32.845877476 +0000
--- tests/A-13-07-M-ok.outhyp	2019-12-12 16:46:02.097977601 +0000
***************
*** 65,71 ****
  5 - Saldar multa de utente
  0 - Sair
  Escolha uma opção: 0 - Chewbacca - chewbacca@milenium.falcon.org - NORMAL - ACTIVO
! 1 - Darth Vader - darth@dark.force.coAm - NORMAL - SUSPENSO - EUR 60
  2 - Luke Skywalker - last.jedi@universe.com - NORMAL - ACTIVO
  3 - Rey - real.last.jedi@universe.com - NORMAL - ACTIVO
  Menu de Gestão de Utentes
--- 65,71 ----
  5 - Saldar multa de utente
  0 - Sair
  Escolha uma opção: 0 - Chewbacca - chewbacca@milenium.falcon.org - NORMAL - ACTIVO
! 1 - Darth Vader - darth@dark.force.coAm - FALTOSO - SUSPENSO - EUR 60
  2 - Luke Skywalker - last.jedi@universe.com - NORMAL - ACTIVO
  3 - Rey - real.last.jedi@universe.com - NORMAL - ACTIVO
  Menu de Gestão de Utentes
***************
*** 83,89 ****
  5 - Saldar multa de utente
  0 - Sair
  Escolha uma opção: 0 - Chewbacca - chewbacca@milenium.falcon.org - NORMAL - ACTIVO
! 1 - Darth Vader - darth@dark.force.coAm - NORMAL - SUSPENSO - EUR 0
  2 - Luke Skywalker - last.jedi@universe.com - NORMAL - ACTIVO
  3 - Rey - real.last.jedi@universe.com - NORMAL - ACTIVO
  Menu de Gestão de Utentes
--- 83,89 ----
  5 - Saldar multa de utente
  0 - Sair
  Escolha uma opção: 0 - Chewbacca - chewbacca@milenium.falcon.org - NORMAL - ACTIVO
! 1 - Darth Vader - darth@dark.force.coAm - FALTOSO - SUSPENSO - EUR 0
  2 - Luke Skywalker - last.jedi@universe.com - NORMAL - ACTIVO
  3 - Rey - real.last.jedi@universe.com - NORMAL - ACTIVO
  Menu de Gestão de Utentes
***************
*** 128,134 ****
  5 - Saldar multa de utente
  0 - Sair
  Escolha uma opção: 0 - Chewbacca - chewbacca@milenium.falcon.org - NORMAL - ACTIVO
! 1 - Darth Vader - darth@dark.force.coAm - NORMAL - SUSPENSO - EUR 25
  2 - Luke Skywalker - last.jedi@universe.com - NORMAL - ACTIVO
  3 - Rey - real.last.jedi@universe.com - NORMAL - ACTIVO
  Menu de Gestão de Utentes
--- 128,134 ----
  5 - Saldar multa de utente
  0 - Sair
  Escolha uma opção: 0 - Chewbacca - chewbacca@milenium.falcon.org - NORMAL - ACTIVO
! 1 - Darth Vader - darth@dark.force.coAm - FALTOSO - SUSPENSO - EUR 25
  2 - Luke Skywalker - last.jedi@universe.com - NORMAL - ACTIVO
  3 - Rey - real.last.jedi@universe.com - NORMAL - ACTIVO
  Menu de Gestão de Utentes
***************
*** 146,152 ****
  5 - Saldar multa de utente
  0 - Sair
  Escolha uma opção: 0 - Chewbacca - chewbacca@milenium.falcon.org - NORMAL - ACTIVO
! 1 - Darth Vader - darth@dark.force.coAm - NORMAL - ACTIVO
  2 - Luke Skywalker - last.jedi@universe.com - NORMAL - ACTIVO
  3 - Rey - real.last.jedi@universe.com - NORMAL - ACTIVO
  Menu de Gestão de Utentes
--- 146,152 ----
  5 - Saldar multa de utente
  0 - Sair
  Escolha uma opção: 0 - Chewbacca - chewbacca@milenium.falcon.org - NORMAL - ACTIVO
! 1 - Darth Vader - darth@dark.force.coAm - FALTOSO - ACTIVO
  2 - Luke Skywalker - last.jedi@universe.com - NORMAL - ACTIVO
  3 - Rey - real.last.jedi@universe.com - NORMAL - ACTIVO
  Menu de Gestão de Utentes
