*** tests/A-13-01-M-ok.out	2019-12-03 17:06:33.560740065 +0000
--- tests/A-13-01-M-ok.outhyp	2019-12-07 22:19:27.714935163 +0000
***************
*** 89,96 ****
  5 - Saldar multa de utente
  0 - Sair
  Escolha uma opção: 0 - Chewbacca - chewbacca@milenium.falcon.org - NORMAL - ACTIVO
! 1 - Darth Vader - darth@dark.force.coAm - FALTOSO - SUSPENSO - EUR 90
! 2 - Luke Skywalker - last.jedi@universe.com - NORMAL - SUSPENSO - EUR 25
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
--- 89,96 ----
  5 - Saldar multa de utente
  0 - Sair
  Escolha uma opção: 0 - Chewbacca - chewbacca@milenium.falcon.org - NORMAL - ACTIVO
! 1 - Darth Vader - darth@dark.force.coAm - FALTOSO - ACTIVO
! 2 - Luke Skywalker - last.jedi@universe.com - FALTOSO - SUSPENSO - EUR 25
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
***************
*** 98,104 ****
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
  3 - Listar utentes
--- 98,105 ----
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Saldar multa de utente: Operação inválida: O utente 1 não se encontra suspenso.
! Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
  3 - Listar utentes
***************
*** 107,113 ****
  0 - Sair
  Escolha uma opção: 0 - Chewbacca - chewbacca@milenium.falcon.org - NORMAL - ACTIVO
  1 - Darth Vader - darth@dark.force.coAm - FALTOSO - ACTIVO
! 2 - Luke Skywalker - last.jedi@universe.com - NORMAL - SUSPENSO - EUR 25
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
--- 108,114 ----
  0 - Sair
  Escolha uma opção: 0 - Chewbacca - chewbacca@milenium.falcon.org - NORMAL - ACTIVO
  1 - Darth Vader - darth@dark.force.coAm - FALTOSO - ACTIVO
! 2 - Luke Skywalker - last.jedi@universe.com - FALTOSO - SUSPENSO - EUR 25
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
***************
*** 124,130 ****
  0 - Sair
  Escolha uma opção: 0 - Chewbacca - chewbacca@milenium.falcon.org - NORMAL - ACTIVO
  1 - Darth Vader - darth@dark.force.coAm - FALTOSO - ACTIVO
! 2 - Luke Skywalker - last.jedi@universe.com - NORMAL - ACTIVO
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
--- 125,131 ----
  0 - Sair
  Escolha uma opção: 0 - Chewbacca - chewbacca@milenium.falcon.org - NORMAL - ACTIVO
  1 - Darth Vader - darth@dark.force.coAm - FALTOSO - ACTIVO
! 2 - Luke Skywalker - last.jedi@universe.com - FALTOSO - ACTIVO
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
